### PR TITLE
Spell areatrigger

### PR DESCRIPTION
**Pull request by Traesh**
Add all spell_areatrigger fields to parse expect SpellMiscId

In the future we could also check if Spell have only one "create areatrigger" effect and auto-fill SpellMiscId

Example output :

DELETE FROM `spell_areatrigger` WHERE (`AreaTriggerId`=3153);
INSERT INTO `spell_areatrigger` (`SpellMiscId`, `AreaTriggerId`, `MoveCurveId`, `ScaleCurveId`, `MorphCurveId`, `FacingCurveId`, `DecalPropertiesId`, `TimeToTarget`, `TimeToTargetScale`, `VerifiedBuild`) VALUES
(UNKNOWN, 3153, 0, 0, 0, 0, 0, 0, 600000, 23420); -- SpellId : 158624

### CODE REVIEW

File: WowPacketParser/Enums/UpdateFields.cs [hunk 1]
```diff
@@ -792,16 +792,19 @@ public enum CorpseDynamicField
 
     public enum AreaTriggerField
```
Comment (DDuarte) [inline WowPacketParser/Enums/UpdateFields.cs]: Does the order matter here?

Comment (Traesh) [inline WowPacketParser/Enums/UpdateFields.cs]: Good question, only did this modification to add missing fields like AREATRIGGER_DECAL_PROPERTIES_ID

Comment (joschiwald) [inline WowPacketParser/Enums/UpdateFields.cs]: don't touch it, the correct enum is in 7.x module

Comment (Traesh) [inline WowPacketParser/Enums/UpdateFields.cs]: I need it in Store/Objects/SpellAreaTrigger.cs, how can i get the enum from the 7.x module in this file ?

Not very comfortable with C# yet, "using WowPacketParserModule.V7_1_5_23360.Enums" don't seem to work

Comment (joschiwald) [inline WowPacketParser/Enums/UpdateFields.cs]: forget what i wrote ^^

Comment (Shauren) [inline WowPacketParser/Enums/UpdateFields.cs]: order doesnt matter, they were sorted alphabetically before

**Comment by DoctorKraft** [comment]
Well ! No news about `AREATRIGGER_EXTRA_SCALE_CURVE +5 / +6` and `AREATRIGGER_TIME_TO_TARGET_EXTRA_SCALE` ? Maybe we can add also sniff these values ? For `AREATRIGGER_EXTRA_SCALE_CURVE +5` it's all time the same value because without AT don't have visual :/

https://github.com/TrinityCore/TrinityCore/issues/19078#issuecomment-280170367

**Pull request merged**